src/build.js

```diff
@@ -401,32 +397,44 @@ class Build
     /**
      * ## getDescription
      *
-     * gets the first text description in the content and returns that
+     * gets description from the content and returns that. Either returns full text or only from
+     * the first block
      *
-     * @param {Array} arr array filled w/ content
+     * @param {Array} storyTextArr array filled w/ content
```

Comment:
but it's not a string, it's an array of objects (yes, I thought it would be a string, too ...)